tests/test_misc.py

```diff
@@ -486,6 +496,9 @@ def sign_revocation_txs(stks, deposit):
         unvault_psbt = stk.rpc.getunvaulttx(deposit)["unvault_tx"]
         unvault_psbt = stk.stk_keychain.sign_unvault_psbt(unvault_psbt, child_index)
         stk.rpc.unvaulttx(deposit, unvault_psbt)
+        assert (
+            len(stk.rpc.listvaults(["activating", "active"], [deposit])["vaults"]) == 1
```

Comment:
Why not only `activating` ?

Reply:
Because they may have been polling the signature in the background before.